### ISSUE TITLE
[2019] Rethinking Softmax Cross-Entropy Loss for Adversarial Robustness

### ISSUE THREAD

**Issue by gatheluck** [gatheluck/PaperReading#7]
## 論文リンク
https://arxiv.org/abs/1905.10626
https://openreview.net/forum?id=Byg9A24tvB

## 公開日（yyyy/mm/dd）
2019/09/26
ICLR2020

## 概要
